src/Browser.php

```diff
@@ -725,6 +726,101 @@ public function withResponseBuffer($maximumSize)
         ));
     }
 
+    /**
+     * #### withHeader()
+     *
+     * The `withHeader(string $header, string $value): Browser` method can be used to
+     * add another header.
+     * For example:
```

Comment:
Should be updated to match the suggestions in the README minus the header and the first line (see also other methods).

Reply:
updated the docblock